docs/core/setup/index.md

```diff
@@ -0,0 +1,453 @@
+
+There are different ways to develop .NET Core apps:
+
+::: zone pivot="os-windows"
+
+- [Visual Studio](https://www.visualstudio.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2019)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Command-line with your favorite editor](../tutorials/using-with-xplat-cli.md)
+
+::: zone-end
+
+::: zone pivot="os-linux"
+
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Command-line with your favorite editor](../tutorials/using-with-xplat-cli.md)
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+- [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Command-line with your favorite editor](../tutorials/using-with-xplat-cli.md)
+
+::: zone-end
+
+You can download a standalone installer for each OS from the following links:
+
+- [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+- [.NET Core 2.2 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.2).
+- [.NET Core 2.1 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+
+It's important to understand the [.NET Core support policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) when choosing which version of .NET Core to develop apps for.
+
+## Check if .NET Core installed
+
+It's possible that the .NET Core SDK may already be installed on your OS. The .NET Core SDK might have been installed if you installed a development IDE, such as Visual Studio or Visual Studio for Mac. You can check what is installed by opening a terminal and running the `dotnet --list-sdks` command:
+
+::: zone pivot="os-windows"
+
+```console
+dotnet --list-sdks
+
+2.1.500 [C:\program files\dotnet\sdk]
+2.1.502 [C:\program files\dotnet\sdk]
+2.1.504 [C:\program files\dotnet\sdk]
+2.1.600 [C:\program files\dotnet\sdk]
+2.1.602 [C:\program files\dotnet\sdk]
+2.2.101 [C:\program files\dotnet\sdk]
+3.0.100 [C:\program files\dotnet\sdk]
+```
+
+::: zone-end
+
+::: zone pivot="os-linux"
+
+```bash
+dotnet --list-sdks
+
+2.1.500 [/home/user/dotnet/sdk]
+2.1.502 [/home/user/dotnet/sdk]
+2.1.504 [/home/user/dotnet/sdk]
+2.1.600 [/home/user/dotnet/sdk]
+2.1.602 [/home/user/dotnet/sdk]
+2.2.101 [/home/user/dotnet/sdk]
+3.0.100 [/home/user/dotnet/sdk]
+```
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+```bash
+dotnet --list-sdks
+
+2.1.500 [/usr/local/share/dotnet/sdk]
+2.1.502 [/usr/local/share/dotnet/sdk]
+2.1.504 [/usr/local/share/dotnet/sdk]
+2.1.600 [/usr/local/share/dotnet/sdk]
+2.1.602 [/usr/local/share/dotnet/sdk]
+2.2.101 [/usr/local/share/dotnet/sdk]
+3.0.100 [/usr/local/share/dotnet/sdk]
+```
+
+::: zone-end
+
+You can also see what runtimes are installed with the command `dotnet --list-runtimes`.
```

Comment:
An alternative command to mention here would be `dotnet --info`.  It list out the a lot more info which is often useful.